roles/addons/prometheus_server/readme.rst

```diff
@@ -15,6 +15,21 @@ Alertmanager is available at http://localhost:9093
 
 Node_exporter is available at http://localhost:9100
 
+In order to allow the role to deploy, you need to add the following 
+content under a file inventory/group_vars/all/addons/monitoring.yml :
```

Comment:
I would rather prefer to set defaults/main.yml, which would allow the role to deploy out-of-the-box.

Reply:
Deal !